### PR TITLE
fix #7109 Impact-T app

### LINKED ISSUE #7109: End-to-end Impact-T application

### PR DESCRIPTION
**Pull request by moellep**

### CODE REVIEW

**Comment by gurhar1133** [comment]
When I try to run the sim I get this error:
```
sirepo.util.UserAlert: UserAlert(error=Simulation library file "SOLRF-filename.rfdata102.txt" does not exist)
```

**Comment by moellep** [comment]
I've added the missing lib files now.

Also, you will need to install a stable version of lume-impact for testing:

```bash
mkdir ChristopherMayes
cd ChristopherMayes
gcl lume-impact
git checkout 5593a87
pip install -e .
```

File: sirepo/template/impactt.py [hunk 1]
```diff
@@ -1,37 +1,302 @@
...
+        percentComplete=100,
+        frameCount=1 if len(r) else 0,
+        reports=r,
+    )
+
+
+def bunch_plot(model, run_dir, frame_index, filename):
+    p = pmd_beamphysics.ParticleGroup(str(run_dir.join(filename)))
+
+    x = p[model.x]
```
Comment (gurhar1133) [inline sirepo/template/impactt.py:83]: looks like`x` and `y` are not referenced anywhere else. 
maybe 
```
values=[x, y],
```
could be
```
values=[p[model.x], p[model.y]],
```

**Review by gurhar1133 — COMMENTED** [review]
Looks really good. Left one comment

**Pull request merged**